File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.py
class Solution(object):
    def reverseWords(self, s):
        """
        :type s: str
        :rtype: str
        """
        
        test = s.split(" ")
        result = ""
        for t in test:
            if t:
                result = t + " " + result
        return result[0:len(result)-1]

File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.java
class Solution {
    public String reverseWords(String s) {
        String[] words = s.split(" ");
        
        System.out.println(Arrays.toString(words));
        
        String answer = "";
        
        for(String x: words){
            if(x != ""){
            answer = x + " " + answer; 
        }}
        
        return answer.substring(0, answer.length() - 1);
    }
}